content/guides/applications/custom-apps/jwt-setup.md

```diff
@@ -112,59 +104,60 @@ openssl rsa -in private.pem -outform PEM -pubout -out public.pem
  Windows users can install and use the [Cygwin][cygwin] package to run OpenSSL.
 </Message>
 
-Next, head over to your application in the [developer console][devconsole] and
-select the"Configuration" link from the left sidebar in your application
-and scroll down to the "Add and Manage Public Keys" section.
+Then, navigate to configuration tab for your application in the
```

Comment:
Recommend changing "navigate to configuration tab..." to "navigate to **the** configuration tab..."